src/globals.css

```diff
@@ -1,3 +1,32 @@
-@tailwind base;
+div.lonboard {
+  color: #11181c;
+  color: hsl(var(--nextui-foreground));
+  background-color: #fff;
+  background-color: hsl(var(--nextui-background));
+  margin: 0;
+  line-height: 1.5;
+  -webkit-text-size-adjust: 100%;
+  tab-size: 4;
+  font-family:
+    ui-sans-serif,
+    system-ui,
+    -apple-system,
+    Segoe UI,
+    Roboto,
+    Ubuntu,
+    Cantarell,
+    Noto Sans,
+    sans-serif,
+    "Apple Color Emoji",
+    "Segoe UI Emoji",
+    Segoe UI Symbol,
+    "Noto Color Emoji";
+  font-feature-settings: normal;
+  font-variation-settings: normal;
+  -webkit-tap-highlight-color: transparent;
```

Comment:
@Bjoern-Rapp it didn't notice this when I reviewed yesterday. Could you explain why this was added?

Reply:
Some selectors in the preflight stylesheet targets html, body and :root elements. They become useless when scoped to the div.lonboard wrapper, since there are no html or body elements inside the wrapper. The only alternative then is to copy / duplicate those properties from the resulting stylesheet and apply them directly to the wrapper.